.github/workflows/build-docker.yml

```
@@ -0,0 +1,88 @@
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+        with:
+          ref: ${{ github.event.inputs.branch || 'master' }}
+
+      - name: Download frontend build artifacts
+        uses: actions/download-artifact@v4
+        with:
+          name: frontend-build
+          path: frontend/dist
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)